### PR TITLE
DBot Indicator Enrichment - Generic

### PR DESCRIPTION
**Pull request by JonathanMeler**
dbot_indicator_enrichment_-_generic_thu_jan_11_2018

### CODE REVIEW

**Comment by asafshen** [comment]
@JonathanMeler 
we have made some changes in playbook structure - please download the playbook again from an updated demisto env (demo2/your local env)

**Comment by JonathanMeler** [comment]
@liorkol test run at https://demo2.demisto.com/#/WarRoom/37805

File: Scripts/script-GetIndicatorDBotScore.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  ec = {}
+  ec["DBotScore"] = []
+
+  for entry in data:
+      indicator_type = entry["indicator_type"]
+      score = entry["score"]
+      source = entry["source"]
+
+      ec["DBotScore"].append({ "Indicator" : indicator,
```
Comment (JonathanMeler) [inline Scripts/script-GetIndicatorDBotScore.yml]: format

File: Scripts/script-GetIndicatorDBotScore.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                              "Type": INDICATOR_TYPE_DICTIONARY[indicator_type],
+                              "Vendor": source,
+                              "Score": score})
+
+  md = tableToMarkdown("Indicator DBot Score",ec["DBotScore"])
+
+  demisto.results({"Type" : entryTypes["note"],
```
Comment (JonathanMeler) [inline Scripts/script-GetIndicatorDBotScore.yml]: format

**Pull request merged**